api/agentpb/agent.proto

```diff
@@ -127,6 +128,22 @@ message ActionResultRequest {
 // ActionResultResponse is an ServerMessage for ActionResultRequest acceptance.
 message ActionResultResponse {}
 
+// CheckConnectionRequest is a ServerMessage asking pmm-agent to check connection with service.
+message CheckConnectionRequest {
+  // Service type.
+  inventory.ServiceType type = 1;
+  // DSN for the service.
+  string dsn = 2;
+}
+
+// CheckConnectionResponse is an AgentMessage containing a result of check request.
+message CheckConnectionResponse {
+  // Successfully connected to the service
+  bool success = 1;
```

Comment:
@BupycHuk, btw, I think `error != ""` would be enough. We don't want an empty error message, and we don't have a case where we can't make one if connection check failed.